lib/grape/validations/validators/values.rb

```diff
@@ -7,6 +7,7 @@ def initialize(attrs, options, required, scope)
       end
 
       def validate_param!(attr_name, params)
+        return unless params.is_a?(Hash)
```

Comment:
Actually I take it back. The parent is the problem, we don't get deep enough to care about the values validator.